src/util/StringUtil.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Splits a string by comma.
+ *
+ * @param input - String instance to split.
+ *
+ * @returns A String array containining the split parts.
+ */
+export function splitCommaSeparated(input: string): string[] {
+  return input.split(/\s*,\s*/u);
+}
+
+/**
+ * Sanitizes part of a URL by replacing non-word content with a '-'.
+ *
+ * @param urlPart - The URL part to sanitize.
+ * @returns The sanitized output.
+ */
+export function sanitizeUrlPart(urlPart: string): string {
```

Comment:
```suggestion
export function sanitizeName(urlPart: string): string {
```
I can see us also using something like this to generate file names for example.